pkg/client/retry.go

```diff
@@ -18,12 +18,29 @@ package client
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/retry"
 	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// MinuteBackoff is a retry.DefaultBackoff that retries for at least a minute (60000ms) but no more than 2 minutes (120000ms).
+var MinuteBackoff = func() wait.Backoff {
+	mb := retry.DefaultBackoff
+	// TotalDuration = 0ms + 10ms + 50ms + 250ms + 1250ms + 6250ms + 31250ms + 60000ms = 99,060 ms > 1 minute
+	// 7 steps
+	mb.Steps = 7
```

Comment:
@kaovilai sure -- and if we're configuring `Cap` anyway, then really this can probably be a much larger number. Step 6 will be 30 seconds after step 5, 7 will be 150 seconds after step 6, 8 will be 750 seconds after step 7, etc. -- but it should stop once it gets to `Cap` in any case.

Reply:
No. Cap here defines how big a step can get.. not the maximum time it will backoff

Reply:
@kaovilai Oh, you're right. I completely misread that when I was looking at the field-level documentation. I guess there's not a field for max time from beginning, so if we did make it configurable, we'd either need to ask users for #steps, or for a total time, and we'd calculate #steps from there.